schemas/com.iterable/system_webhook/jsonschema/1-0-0

```diff
@@ -0,0 +1,695 @@
+            "null"
+          ],
+          "description": "Reference to the email associated with the event (e.g., 'c59667:t93849:docs@iterable.com')",
+          "maxLength": 4096
+        },
+        "emailIds": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "type": "string",
+            "maxLength": 4096,
+            "description": "Reference to email"
+          },
+          "description": "References to emails associated with the event"
+        },
+        "emailListIds": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "type": [
+              "integer",
+              "null"
+            ],
+            "minimum": 0,
+            "maximum": 9223372036854775807
+          },
+          "description": "Lists that a user is subscribed to"
+        },
+        "emailSubject": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Subject of the email associated with the event",
+          "maxLength": 998
+        },
+        "experimentId": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 9223372036854775807,
+          "description": "Reference to the experiment used if the event is an experiment"
+        },
+        "expiresAt": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Expires at (e.g., '2019-08-08 22:37:40 +00:00')",
+          "maxLength": 255
+        },
+        "firstName": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "First name",
+          "maxLength": 4096
+        },
+        "fromPhoneNumber": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Phone number which the event is from (e.g., '+16503926753')",
+          "maxLength": 255
+        },
+        "fromPhoneNumberId": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 9223372036854775807,
+          "description": "Reference to the phone number which the event is from (e.g., 258)"
+        },
+        "fromSMSSenderId": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 9223372036854775807,
+          "description": "Reference to the SMS sender which the event is from (e.g., 258)"
+        },
+        "imageUrl": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Image URL of the event.",
+          "maxLength": 4096
+        },
+        "inAppBody": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "In app body HTML",
+          "maxLength": 32700
```

Comment:
Could this be gigantic? I'm not really sure what it looks like in practice. I worry about either loading HUGE strings or rejecting events because this could be massive. 

If it could be gigantic, then we could exclude it? additionalProperties would capture it (although likely not load it, warehouse depending).

Reply:
I'm actually not sure how big it will be. It might not be that huge as I assume it is being sent in in-app messages that are presented to users in as pop-ups. It could be kind of important is it contains the message that is being displayed. On the other hand, not sure if anyone would want to analyse it in the Snowplow data...

Reply:
I think I'll remove it for now and we can introduce it in future schema versions if necessary.